packages/button/addon/templates/components/nucleus-button.hbs

```diff
@@ -1,4 +1,12 @@
-{{#if icon}}{{nucleus-icon name=icon size=_iconSize}}{{/if}}
+{{#if icon}}
+  {{#if iconOnly}}
+    {{nucleus-icon name=icon size=_iconSize}}
+  {{else}}
+    <span class="icon-holder">
+      {{nucleus-icon name=icon size=_iconSize}}
+    </span>
```

Comment:
Refactor this piece of code such that there's only one nucleus-icon invocation. Perhaps always have the span class icon-holder with an optional `.active` class based on `iconOnly` property.